lib/elixir/lib/base.ex

```diff
@@ -698,7 +698,7 @@ defmodule Base do
 
   defp remove_ignored(string, nil), do: string
   defp remove_ignored(string, :whitespace) do
-    for <<c::8 <- string>>, not c in '\s\t\r\n', into: <<>>, do: <<c::8>>
+    for <<c::8 <- string>>, c not in '\s\t\r\n', into: <<>>, do: <<c::8>>
```

Comment:
Since we're touching this line, it could be great opportunity to get rid of single-letter variable.